src/assets/config.localhost.json

```diff
@@ -1,4 +1,4 @@
 {
   "isEmbeddedMode": false,
-  "splineConsumerApiUrl": "/consumer-localhost"
+  "splineConsumerApiUrl": "http://localhost:8080/consumer"
```

Comment:
The app itself consider that all config can be found in config.json. If we want to configure the app with env vars we should remove config.json at chabge the app init logic to consider just env config.

Reply:
@wajda  I will mark that conversation as resolved. If we decide to change the app init logic, it will be done in a separate task, as it will produce a batch of changes.